akka-docs/src/main/paradox/actors.md

```diff
@@ -2,6 +2,8 @@
 
 @@include[includes.md](includes.md) { #actor-api }
 
+@@project-info{ projectId="akka-actor" }
+
 ## Dependency
```

Comment:
one thought is if we should combine the Dependency section with the project info?
could it even be part of the sbt plugin?
otherwise we could maybe just have it right after or before the project info box (without making it a `##` header?

Reply:
Yes, makes sense to have it built-in there (but I think we can do that in another step https://github.com/lightbend/sbt-paradox-project-info/issues/12). Removing the `## Dependency` might require to update links with that anchor.

Reply:
ok, let's leave it as is for now